x/liquidity/spec/03_state_transitions.md

```diff
@@ -1,174 +1,235 @@
-<!--
-order: 3
--->
+<!-- order: 3 -->
 
-# State Transitions
+ # State Transitions
+
+Messages (Msg) trigger state transitions.
 
 ## Coin Escrow for Liquidity Module Messages
 
-Three messages on the liquidity module need prior coin escrow before confirmation, which causes state transition on `Bank` module. Below lists are describing coin escrow processes for each given message type.
+Transaction confirmation causes state transition on `Bank` module. Some messages on the Liquidity Module require coin escrow before confirmation.
```

Comment:
again, we should link to the bank module and say something about why we mention the bank module